collect_app/src/main/res/values/dimens.xml

```diff
@@ -1,38 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- These can likely all be replaced by Material resources -->
```

Comment:
I got pretty confused about what these were and how they were being used so I figured it'd be a good opportunity to clean up the unused ones. These were introduced at some point on the way to greater standardization but now we can use material resources.